abi.js

```diff
@@ -48,7 +48,7 @@ fastify.get("/get-blockchain", async (request, reply) => {
 });
 
 fastify.get("/get-blocks", async (request, reply) => {
-  const index = request.query.index;
+  const index = parseInt(request.query.index);
```

Comment:
its not necassary

Reply:
It is necessary because by the time I submitted the pr, the chain had not produced any block for more than an hour, and because of the wrong type of js, I could not get the last few blocks to check if there was a problem with the difficulty.